### DISCUSSION TITLE
How to disable logs on jwt.Parse()?

### DISCUSSION THREAD

**Discussion by kayvonkhosrowpour** [lestrrat-go/jwx#788] [Q&A] [answered]
Hi,

We recently moved over to the new v2 library and started getting logs in the following format whenever we call `jwt.Parse()`:
```
2022/08/03 19:27:18 now = 2022-08-03 19:27:18 +0000 UTC, ttv = 2022-08-03 19:07:57 +0000 UTC, skew = 0s, trunc = 1s
```

I haven't found where this log sits in the source code and haven't found a way to disable them. How can I turn these off?

**Comment by isgj** [comment]
This commit https://github.com/lestrrat-go/jwx/commit/ec54ec431aa503ab1d5fc11a5c97a70daf2e0171 removes the log, but it is not tagged, released.

**Comment by lestrrat** [comment, marked as answer]
Sorry, that was a mistake. Please either use HEAD of develop/v2, or wait until I have a release

**Reply by kayvonkhosrowpour** [reply]
What is the ETA on the next release?

**Reply by lestrrat** [reply]
I usually only make releases when I have enough changes accumulated, and so far there's nothing major in the queue, so no releases are planned for a while

**Reply by lestrrat** [reply]
Well, I got a bug report, so a new version has just been shipped today :)
